libs/shared/console-shared/src/lib/enable-box/enable-box.tsx

```diff
@@ -0,0 +1,59 @@
+import { FormEvent, ReactNode, useEffect, useState } from 'react'
+import { InputCheckbox } from '@qovery/shared/ui'
+
+export interface EnableBoxProps {
+  checked: boolean
+  children: ReactNode
+  setChecked: (checked: boolean) => void
+  title: string
+  description: string
+  className?: string
+  name?: string
+  dataTestId?: string
+}
+
+export function EnableBox(props: EnableBoxProps) {
```

Comment:
Why it's not a component on the shared ui ? 
I think it's component of the Design System, we need to have an Storybook for this one

Reply:
We had this way of doing with Gaultier on the v2, the things that was in shared was really the atomic element of our design system. `Button`, `Icon`, `Input`... but as soon as we were starting to mix multiple element together we were putting that in the project shared library.
This component could be indeed be used in other websites, so could be added to our UI library directly yes.